sdcSeeding/Cassandra/cassandraDataGeneration.js

```diff
@@ -0,0 +1,212 @@
+const fs = require('fs');
+const LoremIpsum = require("lorem-ipsum").LoremIpsum;
+
+const lorem = new LoremIpsum({
+  wordsPerSentence: {
+    max: 3,
+    min: 1
+  }
+})
+
+////HELPER FUNCTIONS
+const probability = (probability) => {
+  return Math.random() > (1-probability);
+}
+
+const randomDateWithin3Months = () => {
+  let targetDate = new Date();
+  const positiveOrNegative = Math.random() > 0.5 ? 1 : -1;
+  const numberOfDays = Math.round(Math.random() * 90);
+  targetDate.setDate(targetDate.getDate() + (positiveOrNegative * numberOfDays));
+  return targetDate;
+}
+
+const addRandomDays = (date) => {
+  let newDate = new Date(date);
+  const randomDays = Math.floor(Math.random() * 365);
+  newDate.setDate(newDate.getDate() + randomDays)
+  return newDate;
+}
+
+const zeroPadding = (number, totalNumDigits) => {
+  let string = number.toString();
+  while (string.length < totalNumDigits) {
+    string = '0' + string
```

Comment:
if one of the item is a string, you wont require quotes for 0

Reply:
Good call!